modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/MinusNode.java

```diff
@@ -274,94 +66,21 @@ private void addOnSingle(Row row, int setIdx) {
         }
```

Comment:
this comment should be a few rows above the current one (see line 271).

should we remove the key as far as the first row comes from `setIdx>0` in case all = false?

Reply:
Didn't get about "comment". Can you please clarify what do you mean?
Fixed logic for "all = false"

Reply:
I want to put a comment `should we remove the key as far as the first row comes from setIdx>0 in case all = false?` to the line 271, but GH didn't allow me to do this since this line is too far from the nearest changed by this patch. So I put it to the line 274 and left a note describing what line this intended for (`this comment should be a few rows above the current one (see line 271).` -- this is the note).

Reply:
Sorry if I didn't make myself clear

Reply:
Ok, got it, thank you